src/components/AccountMenuDropdown.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import Menu from './Menu'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+
+class AccountMenuDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      menuActive: false
+    };
+  }
+
+  toggleMenu() {
+    let menuState = !this.state.menuActive;
+    this.setState({
+      menuActive: menuState
+    });
+  }
+
+  render() {
+    let menu
```

Comment:
To make this simpler, I would allow the `Menu` object to determine if it should be viewed or not.  In `Menu`'s render, it could check the `menuActive` prop it will receive:
```jsx
if( this.props.active ) {
  // Render the stuff it currently renders
} else {
  return ""
}
```
This component can now omit all of this checking, and just pass the active prop to the `Menu` component:
```jsx
<ReactCSSTransitionGroup {/* other stuff */}>
  <Menu active={this.state.menuActive} />
</ReactCSSTransitionGroup>
```